docs/helm-chart/production-guide.rst

```diff
@@ -49,9 +49,13 @@ configure the secret name:
 
   data:
     metadataSecretName: mydatabase
+    resultBackendSecretName: mydatabase
 
 .. _production-guide:pgbouncer:
 
+.. warning::
+  `resultBackendSecretName` expects a url that starts with `db+postgresql://`, while `metadataSecretName` expects `postgresql://` or `postgres://` and wont work with `db+postgresql://`. To solve this, it might be necessary to create multiple secrets.
```

Comment:
```suggestion
  If you use `CeleryExecutor`, keep in mind that `resultBackendSecretName` expects a url that starts with `db+postgresql://`, while `metadataSecretName` expects `postgresql://` and wont work with `db+postgresql://`. You'll need to create separate secrets with the correct schema.
```

Starting in 2.3.0, only `postgresql` works, so lets just keep that one. Otherwise, I think this sounds a little better, but feel free to adjust it more.